.github/workflows/artifacts.yml

```diff
@@ -0,0 +1,247 @@
+name: Build and Upload Artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      partner_chains_smart_contracts_sha:
+        description: "partner-chains-smart-contracts commit SHA"
+        default: "38f4c17e0f4052d885c9d438f7fb8ad0b2e7b25e"
```

Comment:
Optional.
Would "v6.2.0" work here? Or maybe some dummy value to not make this revision anything special?
With "v6.2.0" one could easily scan repo for updates when changing flake.nix.

Reply:
This default value is nothing important, it is just the value that is pre-filled in the textbox

I can change to "master", or I can remove entirely

Reply:
v6.2.0 would work if there is a branch named v6.2.0

Reply:
I have removed the default values entirely. User should supply a SHA or a branch when building artifacts